qa/tasks/nvmeof.py

```diff
@@ -54,17 +51,16 @@ def _set_defaults(self):
 
         rbd_config = self.config.get('rbd', {})
         self.poolname = rbd_config.get('pool_name', 'mypool')
-        self.rbd_image_name = rbd_config.get('image_name', 'myimage')
         self.rbd_size = rbd_config.get('rbd_size', 1024*8)
 
         gateway_config = self.config.get('gateway_config', {})
-        conf_vars = gateway_config.get('vars', {})
-        self.cli_image = conf_vars.get('cli_version', 'latest')
-        self.bdev = conf_vars.get('bdev', 'mybdev')
-        self.serial = conf_vars.get('serial', 'SPDK00000000000001')
-        self.nqn = conf_vars.get('nqn', 'nqn.2016-06.io.spdk:cnode1')
-        self.port = conf_vars.get('port', '4420')
-        self.srport = conf_vars.get('srport', '5500')
+        self.namespaces_count = int(gateway_config.get('namespaces_count', 1))
```

Comment:
`rbd_size` doesn't need it because it's passed as string to remote.run anyway, but `namespaces_count` is used for looping through range when creating rbd images. I'll cast it as `int` in the loop instead of here.